uint16 distance0
uint16 distance1
uint16 distance2
uint16 distance3
uint16 distance4
uint16 distance5
